egs/librispeech/ASR/conformer_ctc/train.py

```diff
@@ -124,19 +127,20 @@ def get_params() -> AttributeDict:
     """
     params = AttributeDict(
         {
-            "exp_dir": Path("conformer_ctc/exp"),
+            "exp_dir": Path("conformer_ctc/exp_new"),
```

Comment:
I would suggest to put the exp_dir to the parent folder, e.g. `"exp_dir": Path("exp/conformer_ctc"), Path("exp/tdnn_lstm")`. So we will not copy the exp data when we need a new version model, say, conformer_ctc_v2, confomer_ctc_v3. It is better to keep only source code in the model directory.

Reply:
mm, it was my idea to co-locate the models and code like this.  The idea is, it's easier to keep them in sync, and given a model, it would generally be fairly clear which code generated it.  And we'd add the code to git (at least that was my idea.. although of course that would mean the code would disappear once you switched branches, which might not be ideal.)